src/components/TodoCount.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { useContext } from 'react';
+import { TodosContext } from '../contexts/TodosContext';
+
+export const TodoCount: React.FC = () => {
+  const { todos } = useContext(TodosContext);
+  const todosLeft = todos.filter(todo => !todo.completed).length;
+
+  return (
+    <span className="todo-count" data-cy="todosCounter">
+      {todosLeft === 1
+        ? `${todosLeft} item left`
+        : `${todosLeft} items left`}
```

Comment:
can be more simplify
```suggestion
{`${todosLeft} ${todosLeft === 1 ? 'item' : 'items'} left`}
```